WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsFragment.java

```diff
@@ -702,20 +702,70 @@ public String format(int value) {
         });
     }
 
-    private void showExportContentDialog() {
+    private void requestPurchasesForDeletionCheck() {
+        final Blog currentBlog = WordPress.getCurrentBlog();
+        final ProgressDialog progressDialog = ProgressDialog.show(getActivity(), "", getString(R.string.checking_purchases), true, false);
+        WordPress.getRestClientUtils().getSitePurchases(currentBlog.getDotComBlogId(), new RestRequest.Listener() {
+            @Override
+            public void onResponse(JSONObject response) {
```

Comment:
It might still be a good idea to check to make it future proof. `SiteSettingsFragment` is a beast of a file, we need all the help we can get :)